src/components/Conference.tsx

```diff
@@ -46,17 +47,23 @@ const userMediaConfig = {
     video: true,
 }
 
+// TODO(yunsi): Add data channel config
+// var dataChannelConfig: RTCDataChannelInit = {}
+
 export interface IConferenceState {
     localId: ConfUserID | undefined;
     localStream: MediaStream | undefined;
     remoteStreams: { [id: string]: MediaStream };
+    remoteIsSpeaking: { [id: string]: boolean };
```

Comment:
Currently we just store this information, but we need to add UI to show microphone activity for remote stream based on this.state.remoteIsSpeaking and also support custom rendering for microphone activity.